core/src/main/java/apoc/export/json/ImportJson.java

```diff
@@ -35,8 +43,9 @@
 
                     try (final CountingReader reader = FileUtils.readerFor(fileName);
                          final Scanner scanner = new Scanner(reader).useDelimiter("\n|\r");
-                         JsonImporter jsonImporter = new JsonImporter(importJsonConfig, db, reporter)) {
+                         JsonImporter jsonImporter = new JsonImporter(importJsonConfig, db, reporter, tx)) {
```

Comment:
togli questo, perché lo hai messo?

Reply:
Ok, lo avevo messo per revertare la creazione dei nodi in caso di errore, però se devo verificare solo alle relazioni non serve certo